pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -544,8 +541,9 @@ impl CustomScan for PdbScan {
         } else {
             let results = search_reader.search_minimal(
                 false,
+                search_query_input.contains_more_like_this(),
```

Comment:
```suggestion
                state.custom_state().search_query_input.contains_more_like_this(),
```
to correspond with the above change.